config/main.py

```diff
@@ -3810,25 +3812,27 @@ def feature():
 @feature.command('state', short_help="Enable/disable a feature")
 @click.argument('name', metavar='<feature-name>', required=True)
 @click.argument('state', metavar='<state>', required=True, type=click.Choice(["enabled", "disabled"]))
-def feature_state(name, state):
+@pass_db
+def feature_state(db, name, state):
     """Enable/disable a feature"""
-    state_data = config_db.get_entry('FEATURE', name)
+    state_data = db.cdb.get_entry('FEATURE', name)
```

Comment:
previously, it is using the global config_db. now we are using the db object from the context. 

I think this change is necessary, as in subsequent pr, i will split this big config file into multiple py files, each contain one click group. Then, we need to pass the db around using this context.